bucoffea/helpers/gen.py

```diff
@@ -242,8 +242,9 @@ def setup_gen_jets(df):
         df['nGenJet'],
         pt=df['GenJet_pt'],
         eta=df['GenJet_eta'],
+        abseta=np.abs(df['GenJet_eta']),
         phi=df['GenJet_phi'],
-        mass=0*df['GenJet_pt']
```

Comment:
is this change on purpose? What's it for?

Reply:
No, I don't think the jet mass is used anywhere. I can remove this if you think 0 is better